.idea/runConfigurations/Artemis__Server__Athene_.xml

```diff
@@ -5,9 +5,9 @@
     <option name="ACTIVE_PROFILES" value="dev,bamboo,bitbucket,jira,artemis,scheduling,athene,local" />
     <option name="VM_PARAMETERS" value="--illegal-access=warn" />
     <option name="ALTERNATIVE_JRE_PATH" />
-    <option name="SHORTEN_COMMAND_LINE" value="NONE" />
+    <option name="SHORTEN_COMMAND_LINE" value="MANIFEST" />
```

Comment:
is this change added by mistake?

Reply:
Yep, my Windows machine changes it by default 😠 Will fix it again 🙂